test/test_jit.py

```diff
@@ -1479,6 +1479,44 @@ def getQParamFunc(value):
                    .check_next("dequantize_linear") \
                    .check("conv2d").run(str(scriptModule.graph))
 
+    def test_insert_quantdequant_for_bias(self):
+        class testModule(torch.jit.ScriptModule):
+            def __init__(self):
+                super(testModule, self).__init__()
+                self.conv1 = nn.Conv2d(1, 1, 1, 1).float()
+
+            @torch.jit.script_method
+            def forward(self, x):
+                x = x.quantize_linear(1.0, 0, torch.uint8)
+                x = x.int_repr()
+                x = x.dequantize_linear(1.0, 0, torch.uint8)
+                x = self.conv1(x)
+                return x
+
+        def getQParamFuncW(value):
+            return 'per_tensor_quant', 0.5, 1
+
+        def getQParamFunc(input_scale, weight_scale):
+            scale = 1 / input_scale / weight_scale
+            zero_point = 0
+            return 'per_tensor_quant', scale, zero_point
+
+        scriptModule = testModule()
+
+        torch._C._jit_pass_constant_propagation(scriptModule.graph)
+        torch._C._jit_pass_insert_quantdequant_for_param(scriptModule._c,
+                                                         "forward",
+                                                         "weight",
+                                                         getQParamFuncW)
+        torch._C._jit_pass_insert_quantdequant_for_param(scriptModule._c,
+                                                         "forward",
+                                                         "bias",
+                                                         getQParamFunc)
+
+        FileCheck().check("quantize_linear").check_next("int_repr") \
```

Comment:
I'm not sure what this test actually checks. The original `forward` method already has all the nodes we're checking for, so even if the passes do nothing, the test will pass.

BTW, consider rewriting the test using irparser (you could find an example in `test_pattern_based_rewrite` for instance)- it makes the test more explicit and usually easier to follow.

Reply:
I want to use irparser but we are passing module object down but irparser outputs graph. is it doable to wrap graph in module obj and pass down ?

Reply:
As we discussed, you might want to expose graph-level API for the pass (just for testing) if that would make sense. If not, then it probably should wait until IRParser works on modules as well.

Reply:
yup will do once IRParser work on module level